lapack.go

```diff
@@ -136,3 +136,23 @@ const (
 	InitZ   Comp = 'I'
 	UpdateZ Comp = 'V'
 )
+
+// EigVecSide specifies what eigenvectors will be computed.
+type EigVecSide byte
```

Comment:
GitHub puts the comments here in a non-chronological order for me. Strange.

Anyway, I think I misunderstood you comment, then, @btracey . Do you want me to do anything about the `EigVecSide` type?